### PR TITLE
Add documentation for LOUNGE_HOME environment variable in the CLI helper

### PR DESCRIPTION
**Pull request by astorije**
Couldn't do this in a nicer way, see https://github.com/tj/commander.js/issues/682.

screen shot 2017-08-21 at 01 48 29

(`~` in default not expanded on purpose)

### CODE REVIEW

**Comment by astorije** [comment]
Actually, the default value is wrong, it should be what's in `.lounge_home`. Will fix.

File: src/command-line/utils.js [hunk 1]
```diff
@@ -0,0 +1,38 @@
...
+class Utils {
+	static extraHelp() {
+		[
+			"",
+			"",
+			"  Environment variable:",
+			"",
+			`    LOUNGE_HOME   Path for all configuration files and folders. Defaults to ${colors.green(Utils.defaultLoungeHome())}.`,
+			"",
+		].forEach((e) => console.log(e));
```
Comment (YaManicKill) [inline src/command-line/utils.js]: Do we really have to console log manually? Ew.

Comment (astorije) [inline src/command-line/utils.js]: Yes, unfortunately, see their recommendation in https://github.com/tj/commander.js/issues/682 :(

**Comment by astorije** [comment]
> I feel like we should have a way to not have to add the .on to each file

I tried several ways, and it looks like `.on("--help", ...)` is a per command thing. Which makes sense, in a way, not all commands will have the same helper, but if there was a way to define a global one, that'd be good, except there isn't... :/

**Pull request merged**